.gitignore

```diff
@@ -3,3 +3,4 @@
 **/target
 .vscode/*
 Cargo.lock
+.idea
```

Comment:
Is there any reason for adding this?

Reply:
`.idea` is the Jetbrains IDE generated folder, so add it to `.gitignore`

Reply:
Alright, makes sense :+1: